packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -90,6 +92,20 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     });
   };
 
+  onMouseEnter = () => {
+    this.addHoverBackground();
```

Comment:
can't we do this hover background using css :hover?

Reply:
There is some issue in how the html is structured here I assume: 
> This can't be done with css because we use 2 separate table rows without common parent element.

Maybe could be done with some sibling selectors or something but don't know how the html looks and whether it makes more sense.